views/install.sh.erb

```diff
@@ -112,8 +112,6 @@ elif test -f "/etc/redhat-release"; then
 
   # If /etc/redhat-release exists, we act like RHEL by default
   if test "$platform" = "fedora"; then
-    # FIXME: stop remapping fedora to el
-    # FIXME: remove client side platform_version mangling and hard coded yolo
```

Comment:
Doesn't platform.rb already do this?

https://github.com/chef/opscode-omnitruck/blob/master/platforms.rb#L110-L114